src/checkpoint/checkpoint_txn.c

```diff
@@ -892,12 +892,12 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, bool *trackingp, const char *cfg[
 }
 
 /*
- * __txn_checkpoint_can_skip --
+ * __checkpoint_txn_can_skip --
```

Comment:
I don't find this an improvement and I find it more confusing. Checkpoint was part of the transaction subsystem, which is why functions had `txn_checkpoint` in their names. Reading the new name it seems like it is checking whether it can skip a transaction.

I think I feel that if you really want to make this name change that instead of swapping `checkpoint` and `txn` that you just eliminate the `txn` part of the name. So it becomes `checkpoint_can_skip`, etc.

Reply:
Done.